### PR TITLE
Increased Native Read > 4096 limit

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
Fixes issues with deserialization where data exceeds buffer length.

**Details of Changes**
This PR fixes native `ReadStringFromFile()` where it could only read a limited number of characters. With this PR 4095 bytes is no longer a limit. fyi @gave92

**Validation**
- [x] Built and ran the app

### CODE REVIEW

**Comment by gave92** [comment]
Nice, this this cause issues when reading settings json files?

**Comment by d2dyno1** [comment]
> Nice, this this cause issues when reading settings json files?

I had the same logic in Clipboard Canvas and it did cause issues. Bringing fix for Files too since it's got the same issue.

File: Files/Helpers/NativeFileOperationsHelper.cs [hunk 1]
```diff
@@ -242,29 +242,58 @@ public static bool UnsetFileAttribute(string lpFileName, FileAttributes dwAttrs)
...
                 {
-                    ReadFile(hStream, pBuff, 4096 - 1, &dwBytesRead, IntPtr.Zero);
-                    //str = Encoding.UTF8.GetString(pBuff, dwBytesRead);
+                    using (StreamReader reader = new StreamReader(msBuffer, true))
+                    {
+                        do
+                        {
+                            fixed (byte* pBuffer = buffer)
+                            {
+                                Array.Clear(buffer, 0, buffer.Length);
```
Comment (gave92) [inline Files/Helpers/NativeFileOperationsHelper.cs:276]: I'm fairly sure we can skip this, we only ever read up to "dwBytesRead" so we ain't gonna read garbage, right?

**Review by gave92 — APPROVED** [review]

File: Files/Helpers/NativeFileOperationsHelper.cs (hunk 1 above)
Comment (d2dyno1) [reply Files/Helpers/NativeFileOperationsHelper.cs:276]: I just left it there to indicate that the array is reused.

**Pull request merged**